backend/app/services/repositories/user_repository.py

```
@@ -0,0 +1,90 @@
+        )
+
+        self.db.add(user)
+
+        try:
+            self.db.commit()
+            self.db.refresh(user)
+            return user
+
+        except IntegrityError as e:
+            self.db.rollback()
+            raise ValueError("Email already exists") from e
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)